packages/frontend/core/src/modules/collection-rules/impls/group-by/journal.ts

```
@@ -0,0 +1,32 @@
+    super();
+  }
+
+  groupBy$(
+    _items$: Observable<Set<string>>,
+    _params: GroupByParams
+  ): Observable<Map<string, Set<string>>> {
+    return this.docsService.propertyValues$('journal').pipe(
+      map(values => {
+        const result = new Map<string, Set<string>>();
+        for (const [id, value] of values) {
+          const isJournal = !!value ? 'true' : 'false';
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)